Makefile

```diff
@@ -22,7 +22,7 @@ $(INSTALL_STAMP): pyproject.toml poetry.lock $(PREREQ_STAMP)
 ifdef IN_VENV
 	$(POETRY) install
 else
-	$(POETRY) install --remove-untracked
+	$(POETRY) install --sync
```

Comment:
what does this change? -> is it worth a comment?

Reply:
this is to remove a deprecation warning printed by `poetry`, since `remove-untracked` was renamed in v 1.2.0. The functionality has not changed.

cc @dpp23 for visibility